tests/test_suite_generator/src/linux/exception/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2017 Trail of Bits, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specifi
+
+project(exception_test)
+cmake_minimum_required(VERSION 3.2)
+
+set(PROJECT_CXXFLAGS -Wall -Werror)
+set(PROJECT_LDFLAGS -lpthread)
```

Comment:
Not an error, but judging from the source, you can omit PROJECT_LDFLAGS (this doesn't need pthread)